core/storage/suggestion/gae_models.py

```diff
@@ -665,6 +665,60 @@ def get_in_review_translation_suggestions_by_offset(
             next_offset
         )
 
+    @classmethod
+    def get_reviewable_translation_suggestions_for_single_exploration(
```

Comment:
Nit:
```suggestion
    def get_reviewable_translation_suggestions(
```

There doesn't appear to be any conflicting methods and the "for single exploration" behavior should be clear from the method args.

Reply:
Done